.env.example

```
@@ -1,7 +1,8 @@
-DATABASE_URL="sqlite:schwab.db"
-WS_RPC_URL=wss://base-rpc.publicnode.com
-ORDERBOOK=0x881cf4c0764e733d9C387f3858eE87CcA04AFFe0
-ORDER_OWNER=0xD2843D9E7738d46D90CB6Dff8D6C83db58B9c165
-DEPLOYMENT_BLOCK=34844164
-APP_KEY=<YOUR_SCHWAB_APP_KEY>
-APP_SECRET=<YOUR_SCHWAB_APP_SECRET>
+DATABASE_URL=$DATABASE_URL
+WS_RPC_URL=$WS_RPC_URL
+ORDERBOOK=$ORDERBOOK
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The ORDERBOOK key should go before the WS_RPC_URL key

(UnorderedKey)